.travis.yml

```diff
@@ -12,14 +12,15 @@ script:
   - "rake mysql:test"
 
 rvm:
-  - 2.4.10
+  - 2.4.9
   - 2.5.8
-  - 2.6.6
-  - 2.7.1
+  - 2.6.5
+  - 2.7.0
```

Comment:
Why are these being decremented?

Reply:
At the time of writing, travis was reporting not having access to the newer rubies.  That seems to have been fixed.  I've pushed an updated travis.yml.